mods/linked/moves.js

```diff
@@ -0,0 +1,462 @@
+'use strict';
+
+exports.BattleMovedex = {
+	/**
+	 * Artificial priority
+	 *
+	 */
+	pursuit: {
+		inherit: true,
+		beforeTurnCallback: function (pokemon, target) {
+			let linkedMoves = pokemon.getLinkedMoves();
+			if (linkedMoves.length && !linkedMoves.disabled) {
+				if (linkedMoves[0] === 'pursuit' && linkedMoves[1] !== 'pursuit') return;
+				if (linkedMoves[0] !== 'pursuit' && linkedMoves[1] === 'pursuit') return;
+			}
+
+			target.side.addSideCondition('pursuit', pokemon);
+			if (!target.side.sideConditions['pursuit'].sources) {
+				target.side.sideConditions['pursuit'].sources = [];
+			}
+			target.side.sideConditions['pursuit'].sources.push(pokemon);
+		},
+	},
+	mefirst: {
+		inherit: true,
+		onTryHit: function (target, pokemon) {
+			let action = this.willMove(target);
+			if (action) {
+				let noMeFirst = ['chatter', 'counter', 'covet', 'focuspunch', 'mefirst', 'metalburst', 'mirrorcoat', 'struggle', 'thief'];
+				// Mod-specific: Me First copies the first move in the link
+				let move = this.getMoveCopy(action.linked ? action.linked[0] : action.move);
+				if (move.category !== 'Status' && !noMeFirst.includes(move.id)) {
+					pokemon.addVolatile('mefirst');
+					this.useMove(move, pokemon, target);
+					return null;
+				}
+			}
+			return false;
+		},
+	},
+
+	/**
+	 *	Sucker Punch
+	 *	Will miss on two linked Status moves
+	 *
+	 */
+
+	suckerpunch: {
+		inherit: true,
+		onTry: function (source, target) {
+			let action = this.willMove(target);
+			if (!action || action.choice !== 'move') {
+				this.attrLastMove('[still]');
+				this.add('-fail', source);
+				return null;
+			}
+			if (target.volatiles.mustrecharge && target.volatiles.mustrecharge.duration < 2) {
+				// Duration may not be lower than 2 if Sucker Punch is used as a low-priority move
+				// i.e. if Sucker Punch is linked with a negative priority move
+				this.attrLastMove('[still]');
+				this.add('-fail', source);
+				return null;
+			}
+			if (!target.linked && action.move.category === 'Status' && action.move.id !== 'mefirst') {
```

Comment:
I don't think ``target.linked`` exists. For reference, this is from my XY implementation:

```js
	suckerpunch: {
		inherit: true,
		onTry: function (source, target) {
			let decision = this.willMove(target);
			if (!decision || decision.choice !== 'move') {
				this.add('-fail', source);
				return false;
			}
			if (target.volatiles['mustrecharge'] && target.volatiles['mustrecharge'].duration < 2) {
				// Duration may not be lower than 2 if Sucker Punch is used as a low-priority move
				// i.e. if Sucker Punch is linked with a negative priority move
				this.add('-fail', source);
				return false;
			}
			if (!decision.linked) {
				if (decision.move.category !== 'Status' || decision.move.id === 'mefirst') return;
				this.add('-fail', source);
				return false;
			}

			for (let i = 0; i < decision.linked.length; i++) {
				let linkedMove = this.getMove(decision.linked[i]);
				if (linkedMove.category !== 'Status' || linkedMove.id === 'mefirst') return;
			}
			this.add('-fail', source);
			return false;
		},
	},
```

So it should be ``action.linked`` now I guess?

Reply:
it was target.linked, idk why it got changed

Reply:
Probably a bug then. Apply the patch pls :P
There is another bad instance of ``target.linked`` below too.